.travis.yml

```diff
@@ -1,35 +0,0 @@
-os: linux
-dist: focal
-language: python
-
-
-jobs:
-  include:
-    - name: "Lint code and tests with flake8"
-      python: 3.7
-      script:
-        - flake8 --config=.flake8 src/com2ann.py
-        - flake8 --config=.flake8-tests src/test_com2ann.py
-
-    - name: "Type check code and tests with mypy"
-      python: 3.7
-      script: mypy --python-version=3.8 src/com2ann.py src/test_com2ann.py
-
-    - name: "Run tests on Python 3.8"
-      python: 3.8
-      script: export PYTHONPATH="src"; python -m unittest src/test_com2ann.py
-
-    - name: "Run tests on Python 3.9"
-      python: 3.9
-      script: export PYTHONPATH="src"; python -m unittest src/test_com2ann.py
-
-    - name: "Run tests on Python 3.10"
-      python: "3.10"
-      script: export PYTHONPATH="src"; python -m unittest src/test_com2ann.py
-
-    - name: "Run tests on Python 3.11 (alpha)"
-      python: 3.11-dev
-      script: export PYTHONPATH="src"; python -m unittest src/test_com2ann.py
```

Comment:
Why are you deleting this file? We are currently only running these tests in Travis.

Reply:
> We are currently only running these tests in Travis.

Hm ? I saw the tests running in GitHub actions
https://github.com/ilevkivskyi/com2ann/blob/eb07a3119494af56d3ce96ff41466e17f9f83e2a/.github/workflows/lint_python.yml#L22

https://github.com/ilevkivskyi/com2ann/runs/7389082720?check_suite_focus=true#step:13:1

Also, the tests didn't seem to run in Travis. Given Travis kinda stopped doing OSS, 90% of the time when I see a Travis config on an open source repo, it's an old outdated file, and given that the tests were running in GitHub Actions, it really did look like an oversight ?

But if you want to keep building with Travis, it's your project, I'm not going to argue :D